src/components/Error/Error.tsx

```diff
@@ -0,0 +1,30 @@
+import { FC, useContext } from 'react';
+import classNames from 'classnames/bind';
+import { AppTodoContext } from '../AppTodoContext/AppTodoContext';
+import { ErrorType } from './Error.types';
+
+export const Error: FC = () => {
+  const { errorMessage, setErrorMessage } = useContext(AppTodoContext);
+
+  return (
+    <div
+      className={classNames(
+        'notification is-danger is-light has-text-weight-normal',
+        { hidden: errorMessage === ErrorType.NoError as string },
```

Comment:
Instead of casting `ErrorType.NoError` to `string`, compare `errorMessage` directly to `ErrorType.NoError`. This can prevent potential type errors and improve code readability. 🧐